src/ctl/alloc.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
Please use our global alloc instead of this files

Reply:
Done